server/webapp/WEB-INF/rails.new/webpack/single_page_apps/elastic_profiles.js

```diff
@@ -36,5 +46,5 @@ $(function () {
     );
   };
 
-  Promise.all([PluginInfos.init('elastic-agent')]).then(onSuccess, onFailure);
+  Promise.all([PluginInfos.all()]).then(onSuccess, onFailure);
```

Comment:
can we say `PluginInfos.all().then(onSuccess, onFailure)`?
no need of `Promise.all `